packages/components/organisms/f-header/src/components/Header.vue

```diff
@@ -144,6 +143,9 @@ export default {
 
         headerBackgroundClass () {
             return this.headerBackgroundTheme === 'highlight' ? 'c-header--highlightBg' : '';
+        },
+        showOffersLinkWithContent () {
```

Comment:
Do we need any tests around this computed property?

Reply:
:heavy_check_mark: